packages/mdc-tab-scroller/mdc-tab-scroller.scss

```diff
@@ -59,6 +59,18 @@
   will-change: transform;
 }
 
+.mdc-tab-scroller--start-aligned .mdc-tab-scroller__scroll-content {
```

Comment:
Instead of having all three (start, end, center), could you just have 2 where 1 is the default?

Reply:
The default is that they fill the horizontal space completely with no specific alignment. These three options are all optional.

Reply:
ah gotcha :)